<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFile</class>
 <widget class="QDialog" name="SelectFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Mesh Files</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>1261</width>
     <height>901</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Directory Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="directory_file"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="unselect">
         <property name="text">
          <string>Unselect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back">
         <property name="text">
          <string>Back to Previous Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="select">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Selected Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="selected_file"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="boundary_merger_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>750</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Boundary Face Merger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>750</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="step1_bc_tree">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>321</width>
      <height>701</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>499</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Boundary face list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_boundary_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Boundary Faces</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
